lib/src/base-styles/_nx-tables.scss

```
@@ -268,6 +268,12 @@
   }
 }
 
+//Allows making content visible only to screen readers but not visually
+//Appllied to the <th> when dealing with clickable rows
+.nx-cell--header.nx-cell--row-btn > .nx-cell__row-btn-header-text {
```

It seems likely that the parent part of this selector is no longer necessary
```suggestion
.nx-cell__row-btn-header-text {
```